### DISCUSSION TITLE
MIT 6.042 计算机科学中的数学 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#31] [Announcements]
# MIT 6.042 计算机科学中的数学 | hackway.org

课程名称： Structure and Interpretation of Computer Programs

https://hackway.org/docs/cs/freshman/math/cs6042

**Comment by tom200509** [comment]
修正：课程名称：Mathematics for Computer Science  
课程官网地址： 2015年版：https://ocw.mit.edu/courses/6-042j-mathematics-for-computer-science-spring-2015/
2010年版：https://ocw.mit.edu/courses/6-042j-mathematics-for-computer-science-fall-2010/
2005年版：https://ocw.mit.edu/courses/6-042j-mathematics-for-computer-science-fall-2005/
